packages/lib/services/plugins/Plugin.ts

```diff
@@ -205,7 +208,16 @@ export default class Plugin {
 		return this.contentScriptMessageListeners_[id](message);
 	}
 
+	public addUnloadCleanupCallback(callback: OnUnloadListener) {
```

Comment:
```suggestion
	public addOnUnloadListener(callback: OnUnloadListener) {
```

Use consistent naming.

Reply:
Yes I feel naming could be improved, there are unload/cleanup/listener/callback terms that are swapped depending on the context. Maybe it's all unloadListeners?